test/overlay-dirs.bats

```diff
@@ -0,0 +1,98 @@
+load helpers
+
+function setup() {
+    stacker_setup
+}
+
+function teardown() {
+    cleanup
+    rm -rf recursive bing.ico || true
+}
+
+@test "overlay_dirs works" {
+    require_storage overlay
+    mkdir dir_to_overlay
+    touch dir_to_overlay/file1
+    touch dir_to_overlay/file2
+    touch dir_to_overlay/executable

```

Comment:
Can you add a test here or somewhere that explicitly tests that we do *not* copy owner + group?

Reply:
Good idea, I thought about that, but forgot in the meantime.